### DISCUSSION TITLE
Problem with instalation with yarn

### DISCUSSION THREAD

**Discussion by fabpin** [js-fns/date-fns#3397] [Q&A]
Already i had a proiyect with the following version of node:
node: v19.8.0, npm: 9.5.1 and yarn: 1.22.19, windows 11 pro 64 bits , but already when tried to run the following command "yarn install" appear the following error on the instalation and didnt allow me create de node_modules.
![image](https://user-images.githubusercontent.com/8019419/233382308-0eef953e-2ce7-403a-9c8a-1ab05f1112a6.png)
 NOTE: I will attach the error file to and this command was executed on 3 diferent red: 
[yarn-error.log](https://github.com/date-fns/date-fns/files/11286748/yarn-error.log)

**Comment by joaopedrodcf** [comment]
Did you try to remove node_modules and install again ? Or even delete yarn cache ?
